apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/app/DatabaseManager.java

```diff
@@ -68,7 +71,7 @@
     public TransactionalDataSource getDataSource() {
         if (currentTransactionalDataSource == null || currentTransactionalDataSource.isShutdown()) {
             currentTransactionalDataSource = new TransactionalDataSource(baseDbProperties, propertiesHolder);
-            currentTransactionalDataSource.init(new AplDbVersion());
+            jdbi = currentTransactionalDataSource.init(new AplDbVersion());
```

Comment:
There is not duplication. That code is used in case dynamic adding DB file (+ db initialization) when 'baseDbProperties -> dbFileName' was changed to different value.
Left as is.